### PR TITLE
422 transition from generating triples to triples generated

### PR DESCRIPTION
**Pull request by joke1196**
This PR goals is to allow transition from the event status `generating triples` to `triples generated`

### CODE REVIEW

File: event-log/src/main/scala/io/renku/eventlog/statuschange/commands/ToTriplesGenerated.scala [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import ch.datascience.graph.model.{events, projects}
+import ch.datascience.metrics.LabeledGauge
+import doobie.implicits._
+import eu.timepit.refined.auto._
+import io.renku.eventlog.EventLogDB
+import io.renku.eventlog.statuschange.commands.ProjectPathFinder.findProjectPath
+
+import java.time.Instant
+
+final case class ToTriplesGenerated[Interpretation[_]](
```
Comment (travissarles) [inline event-log/src/main/scala/io/renku/eventlog/statuschange/commands/ToTriplesGenerated.scala:35]: Should this be public?

File: event-log/src/main/scala/io/renku/eventlog/statuschange/commands/ToTriplesGenerated.scala [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    underTriplesGenerationGauge: LabeledGauge[Interpretation, projects.Path],
+    awaitingTransformationGauge: LabeledGauge[Interpretation, projects.Path],
+    now:                         () => Instant = () => Instant.now
+)(implicit ME:                   Bracket[Interpretation, Throwable])
+    extends ChangeStatusCommand[Interpretation] {
+  override def status: events.EventStatus = TriplesGenerated
+
+  override def query: SqlQuery[Int] = SqlQuery(
+    sql"""|UPDATE event 
+          |SET status = $status, execution_date = ${now()}
```
Comment (travissarles) [inline event-log/src/main/scala/io/renku/eventlog/statuschange/commands/ToTriplesGenerated.scala:46]: is execution date the datetime where the status changed?

File: event-log/src/main/scala/io/renku/eventlog/statuschange/commands/ToTriplesGenerated.scala [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+import java.time.Instant
+
+final case class ToTriplesGenerated[Interpretation[_]](
+    eventId:                     CompoundEventId,
+    underTriplesGenerationGauge: LabeledGauge[Interpretation, projects.Path],
+    awaitingTransformationGauge: LabeledGauge[Interpretation, projects.Path],
+    now:                         () => Instant = () => Instant.now
+)(implicit ME:                   Bracket[Interpretation, Throwable])
+    extends ChangeStatusCommand[Interpretation] {
+  override def status: events.EventStatus = TriplesGenerated
```
Comment (travissarles) [inline event-log/src/main/scala/io/renku/eventlog/statuschange/commands/ToTriplesGenerated.scala:42]: Could this be a lazy val?

File: event-log/src/main/scala/io/renku/eventlog/statuschange/commands/ToTriplesGenerated.scala [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+  override def query: SqlQuery[Int] = SqlQuery(
```
Comment (travissarles) [inline event-log/src/main/scala/io/renku/eventlog/statuschange/commands/ToTriplesGenerated.scala:44]: Could this be a lazy val?

File: event-log/src/main/scala/io/renku/eventlog/statuschange/commands/ToTriplesGenerated.scala (hunk 3 above)
Comment (joke1196) [reply event-log/src/main/scala/io/renku/eventlog/statuschange/commands/ToTriplesGenerated.scala:42]: yes sure!

File: event-log/src/main/scala/io/renku/eventlog/statuschange/commands/ToTriplesGenerated.scala (hunk 4 above)
Comment (joke1196) [reply event-log/src/main/scala/io/renku/eventlog/statuschange/commands/ToTriplesGenerated.scala:44]: Yes for the status. but not for the query

File: event-log/src/main/scala/io/renku/eventlog/statuschange/commands/ToTriplesGenerated.scala (hunk 2 above)
Comment (joke1196) [reply event-log/src/main/scala/io/renku/eventlog/statuschange/commands/ToTriplesGenerated.scala:46]: Yes it's when event is updated

File: event-log/src/main/scala/io/renku/eventlog/statuschange/commands/ToTriplesGenerated.scala (hunk 1 above)
Comment (joke1196) [reply event-log/src/main/scala/io/renku/eventlog/statuschange/commands/ToTriplesGenerated.scala:35]: So this is not used yet so yes it could be private

**Review by jachro — APPROVED** [review]

**Pull request merged**